WordPressAuthenticator/Unified Auth/View Related/Get Started/GetStartedViewController.swift

```diff
@@ -36,6 +67,8 @@ class GetStartedViewController: LoginViewController, NUXKeyboardResponder {
     // This is public so it can be set from StoredCredentialsAuthenticator.
     var errorMessage: String?
 
+    var source: SignInSource = .other
```

Comment:
Since I suggested removing `other` case above, maybe we should keep `source` optional here 🤔.

Reply:
Updated in https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/652/commits/8591d33e66b2baf7551b671b4c9adb36443a73fd 👍